RELEASE_PROCESS.md

```diff
@@ -36,13 +36,12 @@ The result of that workflow is the creation and publishing of a Docker image wit
 
 ### Create an official `Release` in GitHub
 
-Once the tag has been pushed to the repository and the build has been verified, a `Release` should be created within GitHub matching the tag.  This is a manual step that must be completed after the automation.
+Once the tag has been pushed to the repository and the build process completed, new `Release` will be created automatically within GitHub matching the tag. Manual step is required to update release notes.
 
-1. Navigate in a browser to [https://github.com/datastax/zdm-proxy/releases](https://github.com/datastax/zdm-proxy/releases) and select the `Draft a new release` button.
-2. Select the `Choose a tag` button and select the previously pushed tag, in our example, `v1.1.0` from the dropdown.
-3. Set the `Release title` to the same name as the tag, in our example, `v1.1.0`.
-4. Paste the contents of the `RELEASE_NOTES` relevant to this release into text-area for `Describe the release`.
-5. Click the `Publish release` button.
+1. Navigate in a browser to [https://github.com/datastax/zdm-proxy/releases](https://github.com/datastax/zdm-proxy/releases).
+2. Search for release matching new tag name and edit it.
```

Comment:
`Search for release matching new tag name and edit it.` --> ` Search for the release matching the new tag name and edit it.`